frontend/src/ErrorPopup/CloseButton.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  Text, TouchableHighlight,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import { SubCardView } from '../common/components';
+import commonColors from '../common/colors';
+
+function CloseButton({
+  onClickClose,
+}) {
+  return (
+    <TouchableHighlight onPress={onClickClose} style={{ padding: 3, width: 100 }}>
+      <SubCardView style={{ backgroundColor: 'rgba(255,0,0,0.7)', justifyContent: 'center' }}>
+        <Text style={{ fontWeight: 'bold', color: commonColors.whiteColor }}>CLOSE</Text>
```

Comment:
the red color of the close button is not the same as the header red in the popup, there is too much color this way, please use the same color